db/schema.rb

```diff
@@ -102,6 +106,11 @@
     t.datetime "updated_at",    null: false
   end
 
+  create_table "canvases", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
I guess we need at least one column that identifies the canvas (maybe the Canvas ID from the manifest). Tempting to use `canvas_id` but it may mess up foreign key assignments, so maybe `iiif_canvas_id`?